src/components/forms/textarea.jsx

```diff
@@ -17,7 +17,7 @@ var TextArea = React.createClass({
         return (
             <FRCTextarea {... this.props}
                          className={classes}
-                         elementWrapperClassName="textarea-group" />
+                         rowClassName="textarea-row" />
```

Comment:
If you would, could you update the styles for all of the input components so it works in the way we want, where the `className` is applied to the top-level element?  It looks like even `Input` ignores `className` right now.